StatePrinter.Tests/ExamplesForDocumentation/ExampleListAndArrays.cs

```diff
@@ -41,23 +41,25 @@ public void ExampleListAndArraysAlternative()
             vendorManager.AddVendor(JobType.JobType3, added3);
 
             var expected = @"new VendorAllocation[]()
-[0] = new VendorAllocation()
 {
-    Allocation = 100
-    Price = 20
-    Share = 20
-}
-[1] = new VendorAllocation()
-{
-    Allocation = 120
-    Price = 550
-    Share = 30
-}
-[2] = new VendorAllocation()
-{
-    Allocation = 880
-    Price = 11
-    Share = 50
+    new VendorAllocation()
```

Comment:
Is there a way we can insert the index in such a way as it reads more natural than in the old format? I have a plan on making stateprinter a tool for helping with debugging states at certain points in the code - and thus seeing the possition may be nice. Thats why they were included.